src/mito2/src/row_converter/sparse.rs

```diff
@@ -42,7 +42,9 @@ struct SparsePrimaryKeyCodecInner {
     // User defined label field
     label_field: SortField,
     // Columns in primary key
-    columns: HashSet<ColumnId>,
+    //
+    // None means all unknown columns is primary key(label field).
```

Comment:
Maybe you should use `label_field`
```suggestion
    // None means all unknown columns is primary key(`Self::label_field`).
```